src/js/components/LineList/index.ts

```diff
@@ -0,0 +1,43 @@
+import Component from "@/core/component";
+import view from "./view";
+import { lineDB } from "@/data";
+import { LineUpdate } from "@/components";
+import { lineService, uiService } from "@/service";
+import { lineStore } from "@/store";
+import { createElement, closest } from "@/utils/dom";
+
+class LineList extends Component {
+  lineUpdate = new LineUpdate();
+
+  protected initDom(): void {
+    this.$container = createElement({
+      tag: "ul",
+      className: "mt-3 pl-0",
+    });
```

Comment:
```suggestion
    this.$container = createElement(`<ul class="mt-3 pl-0"></ul>`);
```
이런 형태로도 만들 수 있도록 createElement를 정의해주면 어떨까요?

Reply:
이렇게하니까 훨씬 깔끔하네용.. 👍 ..한번 구현해보겠습니다!!